### PR TITLE
move raddb packetfence + packetfence-tunnel in conf

### LINKED ISSUE #476: raddb/sites-enabled/packetfence-tunnel should be in conf/radiusd
That file should be in conf/radiusd since it contains configurable content (like eap local account)

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Moves configuration files in sites-enabled into conf so they are easier to manage during the upgrades
# Impacts

Makes these files part of the configuration framework
# Issue

fixes #476 
# Delete branch after merge

YES
# NEWS file entries
## Enhancements
- Moved raddb/sites-enabled/packetfence and raddb/sites-enabled/packetfence-tunnel in conf/radiusd

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -44,6 +44,8 @@ conf/radiusd/sql.conf
 conf/radiusd/proxy.conf.inc
 conf/radiusd/clients.conf.inc
 conf/radiusd/packetfence-cluster
+conf/radiusd/packetfence
+conf/radiusd/packetfence-tunnel
```
Comment (louismunro) [inline .gitignore:48]: In .gitignore? Seems to me we do need to track changes to these.

Edit: Forget it. I am just a little slow today.

**Comment by louismunro** [comment]
LGTM. Let's merge.

**Comment by fdurand** [comment]
debian/packetfence.conffiles ?!

**Comment by julsemaan** [comment]
Everything should be fine now. Merge ?

**Pull request merged**